### PR TITLE
Allow absolute paths for --custom-formatter argument in CLI.

### LINKED ISSUE #2042: Absolute paths don't work in --custom-formatter argument of CLI
Unlike the `--config` argument, the `--custom-formatter` argument only works with relative paths:

```
$ stylelint --custom-formatter=/var/data/formatter.js foo.css

module.js:327
    throw err;
    ^
Error: Cannot find module '/home/username/var/data/formatter.js'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/usr/lib/node_modules/stylelint/dist/cli.js:61:15)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
```

### PR DESCRIPTION
**Pull request by JorisVanEijden**
Fixes #2042

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -115,7 +115,10 @@ const cli = meow(meowOptions, minimistOptions)
 
 let formatter = cli.flags.formatter
 if (cli.flags.customFormatter) {
-  formatter = require(path.join(process.cwd(), cli.flags.customFormatter))
+  var customFormatter = path.isAbsolute(cli.flags.customFormatter)
```
Comment (evilebottnawi) [inline src/cli.js]: Don't use `var`. Use `const` instead.

File: src/cli.js [hunk 2]
```diff
@@ -115,9 +115,9 @@ const cli = meow(meowOptions, minimistOptions)
 
 let formatter = cli.flags.formatter
 if (cli.flags.customFormatter) {
-  var customFormatter = path.isAbsolute(cli.flags.customFormatter)
+  let customFormatter = path.isAbsolute(cli.flags.customFormatter)
```
Comment (evilebottnawi) [inline src/cli.js]: Use `conts`. https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

**Comment by davidtheclark** [comment]
Excellent! Thanks @JorisVanEijden!

Can you please add a test to prevent future regressions? Then we can merge.

**Comment by JorisVanEijden** [comment]
I'm sorry. This was about the limit of my Javascript programming.
I did look for a test for the `--config` argument which had a similar problem (commit 583056d44e740655406867125a813fb36c9ce605) so I could copy & modify that, but there doesn't appear to be one.

**Comment by davidtheclark** [comment]
Oops, my mistake: there aren't tests for the CLI yet. That must be why this happened :(

**Comment by davidtheclark** [comment]
Changelog note:

> -   Fixed: CLI now understands absolute paths for the `--custom-formatter` option.

**Pull request merged**